docs/source/notebooks/04_Emulating_hydrological_models.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # This syntax will allow you yo actually load the data in memory and work with it directly. Note that there is no way to work with the zip-file directly,
</pre><p>These are the actual outputs loaded as Python objects. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/raven/pull/424/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>true, fixed!</p>